src/prefect/orion/schemas/core.py

```diff
@@ -268,6 +271,10 @@ class TaskRunPolicy(PrefectBaseModel):
     retry_delay: Optional[int] = Field(
         default=None, description="The delay time between retries, in seconds."
     )
+    flow_restart_index: Optional[int] = Field(
```

Comment:
why wouldn't this always be equal to `restarts`?

Reply:
if a flow run has been restarted multiple times, some of its task runs will be from the first attempt at running the flow and some other task runs will be from a later attempt